ramls/loginCredentials.json

```diff
@@ -15,6 +15,10 @@
     "password": {
       "type": "string",
       "description": "Password"
+    },
+    "tenant": {
+      "description": "Tenant; can be provided to override the current tenant",
```

Comment:
Would be good to have more description about what overriding the tenant would mean in this context. Consider someone coming to this for the first time. Also avoid use the semicolon.